lib/mix/lib/mix/release.ex

```diff
@@ -786,9 +793,9 @@ defmodule Mix.Release do
   The exact chunks that are kept are not documented and may change in
   future versions.
   """
-  @spec strip_beam(binary()) :: {:ok, binary} | {:error, :beam_lib, term}
-  def strip_beam(binary) do
-    case :beam_lib.chunks(binary, @significant_chunks, [:allow_missing_chunks]) do
+  @spec strip_beam(binary(), List.t()) :: {:ok, binary} | {:error, :beam_lib, term}
+  def strip_beam(binary, exceptions) do
```

Comment:
I think we could use here the syntax for default arguments:

```suggestion
  def strip_beam(binary, exceptions \\ []) do
```

and maybe include some guards?

```suggestion
  def strip_beam(binary, exceptions \\ []) when is_list(exceptions) do
```